WordPress/src/main/java/org/wordpress/android/ui/posts/editor/media/EditorMedia.kt

```diff
@@ -144,7 +126,7 @@ class EditorMedia @Inject constructor(
                     site,
                     freshlyTaken,
                     editorMediaListener,
-                    (editorType == POST_EDITOR) // also start upload if this is a Posts editor
```

Comment:
@mzorz this was removed because it's always true right?

Reply:
Okay, I am seeing where we used to have `STORY_EDITOR` for stories. Will review this in more detail.

Reply:
is it a case where this is not necessary anymore since we aren't sharing the `EditorMedia` with the `StoryComposerActivity` anymore?

Reply:
Right, there's no longer a disntinction in EditorMedia for Stories so, it's always `true` there 👍

Reply:
Okay awesome!